@grpc/grpc-js@0.8.0 does not contain built js files

### Problem description
When attempting to consume `@grpc/grpc-js@0.8.0` from my typescript project i get the following error:

```
Cannot find module '[redacted]/node_modules/@grpc/grpc-js/build/src/index.js'. Please verify that the package.json has a valid "main" entry.
```

This is because package.json main points to a file that does not exist.

https://github.com/grpc/grpc-node/blob/227a35e89955bd5e5098ac3069237f7459b4e341/packages/grpc-js/package.json#L7

```shell
richardwillis:~/[redacted]/node_modules/@grpc/grpc-js 
$ tree
.
├── LICENSE
├── README.md
├── build
│   ├── package.json
│   └── src
│       ├── backoff-timeout.d.ts
│       ├── call-credentials-filter.d.ts
│       ├── call-credentials.d.ts
│       ├── call-stream.d.ts
│       ├── call.d.ts
│       ├── channel-credentials.d.ts
│       ├── channel-options.d.ts
│       ├── channel.d.ts
│       ├── client-interceptors.d.ts
│       ├── client.d.ts
│       ├── compression-filter.d.ts
│       ├── constants.d.ts
│       ├── deadline-filter.d.ts
│       ├── events.d.ts
│       ├── filter-stack.d.ts
│       ├── filter.d.ts
│       ├── http_proxy.d.ts
│       ├── index.d.ts
│       ├── load-balancer-pick-first.d.ts
│       ├── load-balancer-round-robin.d.ts
│       ├── load-balancer.d.ts
│       ├── load-balancing-config.d.ts
│       ├── logging.d.ts
│       ├── make-client.d.ts
│       ├── max-message-size-filter.d.ts
│       ├── metadata.d.ts
│       ├── object-stream.d.ts
│       ├── picker.d.ts
│       ├── resolver-dns.d.ts
│       ├── resolver-uds.d.ts
│       ├── resolver.d.ts
│       ├── resolving-load-balancer.d.ts
│       ├── server-call.d.ts
│       ├── server-credentials.d.ts
│       ├── server.d.ts
│       ├── service-config.d.ts
│       ├── status-builder.d.ts
│       ├── stream-decoder.d.ts
│       ├── subchannel-pool.d.ts
│       ├── subchannel.d.ts
│       └── tls-helpers.d.ts
├── package.json
└── src
    ├── backoff-timeout.ts
    ├── call-credentials-filter.ts
    ├── call-credentials.ts
    ├── call-stream.ts
    ├── call.ts
    ├── channel-credentials.ts
    ├── channel-options.ts
    ├── channel.ts
    ├── client-interceptors.ts
    ├── client.ts
    ├── compression-filter.ts
    ├── constants.ts
    ├── deadline-filter.ts
    ├── events.ts
    ├── filter-stack.ts
    ├── filter.ts
    ├── http_proxy.ts
    ├── index.ts
    ├── load-balancer-pick-first.ts
    ├── load-balancer-round-robin.ts
    ├── load-balancer.ts
    ├── load-balancing-config.ts
    ├── logging.ts
    ├── make-client.ts
    ├── max-message-size-filter.ts
    ├── metadata.ts
    ├── object-stream.ts
    ├── picker.ts
    ├── resolver-dns.ts
    ├── resolver-uds.ts
    ├── resolver.ts
    ├── resolving-load-balancer.ts
    ├── server-call.ts
    ├── server-credentials.ts
    ├── server.ts
    ├── service-config.ts
    ├── status-builder.ts
    ├── stream-decoder.ts
    ├── subchannel-pool.ts
    ├── subchannel.ts
    └── tls-helpers.ts

3 directories, 86 files
```


### Environment
 - OS name, version and architecture: macos catalina
 - Node version: v12.16.1
 - Node installation method: nvm
 - Package name and version: @grpc/grpc-js@0.8.0

--- a/packages/grpc-js/.eslintrc
+++ b/packages/grpc-js/.eslintrc
@@ -5,6 +5,7 @@
     "node/no-unpublished-import": ["error", {
         "tryExtensions": [".ts", ".js", ".json", ".node"]
     }],
-    "@typescript-eslint/no-unused-vars": "off"
+    "@typescript-eslint/no-unused-vars": "off",
+    "node/no-unpublished-require": "off"
   }
 }

--- a/packages/grpc-js/src/index.ts
+++ b/packages/grpc-js/src/index.ts
@@ -62,8 +62,7 @@ import {
   ServerDuplexStream,
 } from './server-call';
 
-import { engines as supportedEngines } from '../package.json';
-const supportedNodeVersions = supportedEngines.node;
+const supportedNodeVersions = require('../../package.json').engines.node;
 if (!semver.satisfies(process.version, supportedNodeVersions)) {
   throw new Error(`@grpc/grpc-js only works on Node ${supportedNodeVersions}`);
 }

--- a/packages/grpc-js/src/subchannel.ts
+++ b/packages/grpc-js/src/subchannel.ts
@@ -29,7 +29,7 @@ import { LogVerbosity } from './constants';
 import { shouldUseProxy, getProxiedConnection } from './http_proxy';
 import * as net from 'net';
 
-import { version as clientVersion } from '../package.json';
+const clientVersion = require('../../package.json').version;
 
 const TRACER_NAME = 'subchannel';
 

